.travis.yml

```diff
@@ -8,6 +8,7 @@ php:
   - 7.3
   - 7.4
   - 8.0
+  - 8.1
 
 sudo: false
 
```

Comment:
@matt-harvey please add below code to this line:

```php
dist: bionic
```

Travis CI seems can't find PHP 8.1 image (on default xenial build environment), it's break the merging process...

Reply:
> @matt-harvey please add below code to this line:
> 
> ```yaml
> dist: bionic
> ```
> 
> Travis CI seems can't find PHP 8.1 image (on default xenial build environment), it's break the merging process...

It's also can't resolve that problem, please [check it out](https://docs.travis-ci.com/user/languages/php/)

Reply:
we need claim that for YAML:

```yaml
arch: amd64
```

Reply:
I will fix CI problem, merge this PR for now.

Reply:
OK, thanks @hidehalo